anti-analysis/anti-forensic/patch-process-command-line.yml

```diff
@@ -8,6 +8,8 @@ rule:
     scope: function
     att&ck:
       - Defense Evasion::Masquerading::Masquerade Task or Service [T1036.004]
```

Comment:
Did we decide that the att&ck mapping should be Defense Evasion::Process Injection [T1055] instead of Masquerading...?